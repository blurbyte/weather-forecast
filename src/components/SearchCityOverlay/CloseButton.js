import React from 'react';

import Hitbox from '../Hitbox';
import { CrossMark } from '../Icons';

const CloseButton = props => (
  <Hitbox {...props} aria-label="Close">
    <CrossMark />
  </Hitbox>
);

export default CloseButton;
