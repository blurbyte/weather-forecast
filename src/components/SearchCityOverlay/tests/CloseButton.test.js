import React from 'react';
import renderer from 'react-test-renderer';

import CloseButton from '../CloseButton';

test('renders correctly', () => {
  const component = renderer.create(<CloseButton />);
  expect(component.toJSON()).toMatchSnapshot();
});
