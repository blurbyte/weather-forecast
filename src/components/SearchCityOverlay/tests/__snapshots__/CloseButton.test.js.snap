// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-bdVaJa gutKeJ"
>
  <svg
    height="24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M13.463 12L23.697 1.766A1.034 1.034 0 1 0 22.234.303L12 10.537 1.766.303A1.034 1.034 0 1 0 .303 1.766L10.537 12 .303 22.234a1.034 1.034 0 0 0 1.462 1.463L12 13.463l10.234 10.234a1.032 1.032 0 0 0 1.463 0 1.034 1.034 0 0 0 0-1.463L13.463 12z"
      fill="currentColor"
    />
  </svg>
</button>
`;
